spec/rollout_spec.rb

```diff
@@ -89,6 +89,23 @@
     end
   end
 
+  ##
+  # CONTRACT: assess feature status for the user
+  #
+  # assumming client uses it like this rollout.active?('feature_name', user)
+  #
+  # accepts feature name and some kind of identifiable object
+  #
+  # returns boolean: true or false
+  # * true when feature is active for the user
```

Comment:
How should I interpret `active?(feature_name, identifiable)` when `identifiable` is not a user? Or should we say in the contract that `identifiable` is always a user?

Reply:
Let's go with "always a user" :)